helm/templates/primary/replicaset-initiate-job.yaml

```diff
@@ -28,14 +28,60 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: metadata.namespace
+            {{- if eq .Values.config.security.authorization "enabled" }}
+            - name: MONGO_AUTHENTICATION_ENABLED
+              value: "true"
+            {{- end }}
+            {{- if or .Values.createUsers (eq .Values.config.security.authorization "enabled") }}
```

Comment:
should it be `and` instead of `or`?

Reply:
In our SaaS clusters we have to create users even though authorization is disabled right.